pcdsdevices/state.py

```diff
@@ -639,7 +641,7 @@ def state_config_dotted_names(state_count: int) -> list[Optional[str]]:
     ]
 
 
-def state_config_dotted_velos(state_count: int) -> list[Optional[str]]:
+def state_config_dotted_velos(state_count: int) -> list[str | None]:
```

Comment:
is `typing.Optional` out of style now?

Reply:
I'm a bit more partial to `list[Optional[str]]` than `list[str | None]`, personally. 

`pyupgrade` uses https://peps.python.org/pep-0604/ wherever possible, though

Reply:
In my head, `Optional` is more intuitive as an argument, but not as a return type, where they are about the same to me.
I'll need to sit on it to form a stronger opinion myself.